1ejer/index.css

```
@@ -0,0 +1,111 @@
+.container{
+    max-width: 1200px;
+    margin: auto;
+    display: flex;
+    gap: 24px;
+    padding: 20px;
+}
+
+.left{      
+    width: 40%;
+    margin-top: 69px;
+    margin-bottom: 90px;
+    display: flex;
+    flex-direction: column;
+}
+
+.title{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 54px;
+    text-align: center;
+    color: #000000;
+    margin-top: auto;
+}
+
+.reference{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    line-height: 1.33;
+    letter-spacing: normal;
+    text-align: center;
+    color: #000000;
+}
+
+.description{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    line-height: 1.25;
+    letter-spacing: normal;
+    text-align: center;
+    color: #000000;
+}
+
+button{
+    width: 260px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 18px;
+    letter-spacing: 1px;
+    text-align: center;
+    color: #ffffff;
+}
+
+.reactivatenow{
+    margin: 24px auto 42px auto;
+    padding: 15px 16px;
+    background-color: #fb6a66;
+    border:none;
+}
+
+.image-container {
+      width: 50%;
+      margin-top: 42px;
+      margin-bottom: 42px;
+      padding-top: 30px;
+      padding-bottom: 20px;
+}
+
+img{
```

Lo mismo que el botón
```suggestion
.ejer1-container .image-container img{
```